docs/sources/alerting/fundamentals/annotation-label/labels-and-label-matchers.md

```diff
@@ -42,3 +44,4 @@ In this situation,
 - A label matcher defined as `foo!=bar` will _not_ match this alert rule.
 - A label matcher defined as `id=~[0-9]+` will match this alert rule.
 - A label matcher defined as `baz!~[0-9]+` will match this alert rule.
+- Two label matchers defined as `foo=bar` and `id=~[0-9]+` will match this alert rule.
```

Comment:
I wonder if this can be written more clearly.

Reply:
I would also delete "In this situation...".

Reply:
Changed.